user.go

```diff
@@ -71,6 +90,25 @@ func (c *Client) UnmuteUser(targetID string, userID string) error {
 	return c.makeRequest(http.MethodPost, "moderation/unmute", nil, data, nil)
 }
 
+// Removes a mute
+// targetID: the user getting un-muted
+// userID: the user muting the target
+func (c *Client) UnmuteUsers(targetIDs []string, userID string) error {
+	switch {
```

Comment:
can't use a switch here (unless you use the fall through but that's ugly)

Reply:
(both params needs validation)

Reply:
switch looks cleaner than many if's, but it depends on reviewer

Reply:
but ofc its not problem to just write multiple `if`.